### PR TITLE
Versions bump

### PR DESCRIPTION
**Pull request by francescopellegrini**

### CODE REVIEW

**Comment by francescopellegrini** [comment]
@NeQuissimus @manub I would create a release after merging this: what do you think?

File: build.sbt [hunk 1]
```diff
@@ -2,7 +2,7 @@ import sbtrelease.Version
 
 val embeddedKafkaVersion = "2.1.0-SNAPSHOT"
```
Comment (NeQuissimus) [inline build.sbt]: You're going to need to change this if you want to cut a release

Comment (manub) [reply build.sbt]: this would be dealt with automatically if using the `sbt-release` plugin!

Comment (NeQuissimus) [reply build.sbt]: If that is so, this LGTM

Comment (francescopellegrini) [reply build.sbt]: I'll release `embedded-kafka` first, then bump the dependency and finally release this library (using `sbt-release`).

**Pull request merged**